lib/Cas/Protocol/SamlValidateResponder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SimpleSAML\Module\casserver\Cas\Protocol;
+
+use SimpleSAML\Configuration;
+use SimpleSAML\XML\Shib13\AuthnResponse;
+
+class SamlValidateResponder
+{
+
```

Comment:
Nitpick, but PSR-2 prescribes us to not have whitespace between the class definition and the first line